plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -226,15 +226,24 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
         )
     }
 
-    private fun sendValidationResultTelemetry(validationResult: ValidationResult) {
-        if (!validationResult.valid) {
+    private fun sendValidationResultTelemetry(validationResult: ValidationResult, projectOnLoad: Boolean = false) {
+        // Old telemetry event to be fired only when users click on transform
+        if (!validationResult.valid && !projectOnLoad) {
```

Comment:
Can we rename `projectOnLoad`? Maybe it's just me but kind of hard to understand what it represents,  seems to be `true` only if we're resuming the job?

Reply:
Its only fired when the project is opened up. Maybe something like `onFirstOpen` or something.